packages/ses/scripts/bundle.js

```diff
@@ -39,6 +39,30 @@ const main = async () => {
   await write('dist/lockdown.umd.min.js', terse);
 };
 
+const makeLibrary = async () => {
+  const bundle = await makeBundle(
+    read,
+    pathToFileURL(resolve('../util.js', import.meta.url)).toString(),
+  );
+  const { code: terse } = terser.minify(bundle, {
+    mangle: false,
+    keep_classnames: true,
+  });
+
+  console.log(`Util Bundle size: ${bundle.length} bytes`);
+  console.log(`Util Minified bundle size: ${terse.length} bytes`);
+
+  await fs.promises.mkdir('dist', { recursive: true });
+  // we only write out cjs bundles due to compartment-mapper bundler limitation
```

Comment:
I don’t understand this limitation. Compartment mapper should be able to slurp up the original ESM.

Reply:
it does handle consuming the esm just fine but bundles are not currently able to export as esm. if you evaluate the bundle, it will return the namespace, but will not create export statements. we want to be able to import the utilities normally. we're able to do it in commonjs by a sort of hack. this will result in the bundle importable as a commonjs module  https://github.com/endojs/endo/blob/558a3c3d8d3a8ae7518ed1dbb9e23e7c0fb8e8c0/packages/ses/util.js#L26-L31

Reply:
if we dont need to bundle, we dont need this workaround.

Reply:
Can you at least file an issue for how we could fix the inability to produce ESM exports?  Then, please reference it here (and other places that would ideally be unnecessary) in a comment as `TODO: solve this workaround by fixing #XXXX`.